dev/badges.md

```diff
@@ -125,3 +127,4 @@ A collection of unsorted ideas. Move one of these into "To Be Implemented" if yo
 - Back again: Be mentored for the second time
 - Third time's the charm: Be mentored for the third time
 - Hold a conference talk about Exercism
+- Report a non-trivial security vulnerability in relation to Exercism in an ethical way
```

Comment:
non-trivial is meant to exclude random bullshit reports in a dev dependency 5 layers down based on an npm advisory.